JASP-Desktop/data/datasetloader.cpp

```diff
@@ -77,6 +78,8 @@ void DataSetLoader::loadPackage(const string &locator, const string &extension,
 
 void DataSetLoader::syncPackage(const string &locator, const string &extension, boost::function<void(const string &, int)> progress)
 {
+	Utils::sleep(100);
+
```

Comment:
Fortunately I did not have to write this code!

Reply:
I really wished it wasn't necessary, but I can't think of anything better...